src/GeneralStateTestsFiller/stEIP3670/CreateTransactionEOF1_TerminatingOpcodesFiller.yml

```diff
@@ -0,0 +1,279 @@
+CreateTransactionEOF1_TerminatingOpcodes:
```

Comment:
Terminating instructions are not required now, so maybe this file (and similar others for CREATEs can be removed)

Reply:
removed this and similar files testing terminating opcodes